bin.src/validateCfht.py

```diff
@@ -25,15 +25,12 @@
 import os.path
 import sys
 
-from lsst.validate.drp import checkAstrometry
```

Comment:
`validateCfht.py` and `validateDecam.py` are very nearly the same thing modulo some numbers. Do they really need to be completely separate pieces of code? It would seem more natural to have a `ValidateTask` containing all the boilerplate and with subclasses for CFHT and DECam (and other instrumentation in future, one assumes), or even to have a single validation task with appropriate configuration parameters so it can address different instruments.

(I have effectively the same comments on both pieces of code, and won't repeat them.)